docs/standard/containerized-lifecycle-architecture/Docker-application-lifecycle/containers-foundation-for-devops-collaboration.md

```diff
@@ -51,15 +77,15 @@ DevOps should be considered a journey, not a destination. It should be implement
 
 Here are some of the most important benefits provided by a solid DevOps workflow:
 
--   Deliver better-quality software, faster and with better compliance
+- Deliver better-quality software, faster and with better compliance
```

Comment:
nit: lists with long sentences like this should have punctuation
https://docs.microsoft.com/en-us/style-guide/scannable-content/lists#punctuation